Source/Core/Core/AchievementManager.cpp

```diff
@@ -1583,6 +1619,9 @@ void AchievementManager::HandleLeaderboardTriggeredEvent(const rc_runtime_event_
 {
   const auto event_id = runtime_event->id;
   const auto event_value = runtime_event->value;
+  for (auto itx = m_active_leaderboards.begin(); itx < m_active_leaderboards.end(); itx++)
```

Comment:
This seems like it can be collapsed to:

```cpp
std::erase_if(m_active_leaderboards,
              [event_id](const auto& entry) { return entry.id == event_id; });
```